sysutils/fx/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        antonmedv fx 18.0.1
+
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         Command-line tool and terminal JSON viewer
+long_description    ${description}
+
+checksums           rmd160  86d17a60e47f88a334292566ae584af06c08343e \
+                    sha256  a0e6bf4bac205a5b04b9553d47692a6b80b9fc34e2b679c8472b70686ca78cd5 \
+                    size    11106322
+
+master_sites        ${github.homepage}/releases/download/${version}
+distname            ${name}-macos
+
+extract.mkdir       yes
+
+use_configure       no
+use_zip             yes
```

Comment:
The github portgroup won't figure anything out; it will do what you tell it to. In this case, the port is downloading a manually-created file from the github project's releases section. There is no predicting what filenames or compression formats developers will use for such files so the port must specify. In this case, the port correctly specifies that this particular file is a zip file.